microsoft-edge/web-platform/release-notes/136.md

```diff
@@ -308,37 +354,52 @@ The new violation called _Potential Permissions Policy violation_ analyzes:
 
 Based on the above, the violation detects the conflict between the enforced Permissions Policy and the permissions being propagated to iframes.
 
+<!-- todo: link?
+See [__](__) at __.
+-->
+
 
 <!-- ---------- -->
 ###### Predictable reported storage quota
 
 The StorageManager's `estimate`  now reports a predictable storage quota for sites that have limited storage permissions.
 
-This prevents the detection of a user's browsing mode via the storage API by reporting an artificial quota, equal to the current usage + 10 Gib, in all browsing modes.
+This prevents the detection of a user's browsing mode via the storage API by reporting an artificial quota.  The quota is the current usage + 10 Gib,<!-- todo: GiB?  GB?  spell out --> in all browsing modes.
```

Comment:
"Gib" is evidently wrong capzn

https://www.bing.com/search?q=abbreviation+gib - GiB = gibibyte or Gibibyte
https://www.bing.com/search?q=abbreviation%20gigabit - GB = Gigabit